src/Mesh/babylon.mesh.ts

```diff
@@ -51,6 +51,7 @@
         public _binaryInfo: any;
         private _LODLevels = new Array<Internals.MeshLODLevel>();
         public onLODLevelSelection: (distance: number, mesh: Mesh, selectedLevel: Mesh) => void;
+        public highlight = false;
```

Comment:
Ok... well my Git post put somethings in that i use that weren't suppose to be uploaded to master. But i will let you know what this did for me in case you're interested in it in the full build.
You have renderOutline. Which is quite nice. But i was looking for something that would highlight the mesh instead. So i just added my own function that highlights a mesh: in mesh.ts:

if (this.highlight){
                scene.getOutlineRenderer().render(subMesh, batch);
            }